ibtk/include/ibtk/LDataManager.h

```diff
@@ -896,6 +896,13 @@ class LDataManager : public SAMRAI::tbox::Serializable, public SAMRAI::mesh::Sta
      */
     void putToDatabase(SAMRAI::tbox::Pointer<SAMRAI::tbox::Database> db);
 
+    /*!
+     * Register user defined Lagrangian data to be maintained
+     *
+     */
+    void registerUserDefinedLData(const std::string& data_name,
+                                  int depth);
```

Comment:
default value of 1?

Reply:
I think we should force user to provide depth, so that they don't accidentally use depth of 1. But I am also fine with a default depth of 1. 

(I think I do not like default arguments in general. So I am biased :-( )